src/plugins_types.h

```diff
@@ -77,19 +80,21 @@ struct lysp_module;
  * required format via ::ly_type_print_clb implementation.
  */
 
-/**
- * @brief Helper function for various plugin functions to generate error information structure.
+/*
+ * @brief Create and fill error structure.
+ *
+ * Helper function for various plugin functions to generate error information structure.
  *
- * @param[in] level Error level of the error.
- * @param[in] code Code of the error.
- * @param[in] vecode Validity error code in case of LY_EVALID error code.
- * @param[in] msg Error message.
- * @param[in] path Path to the node causing the error.
- * @param[in] apptag Error-app-tag value.
+ * @param [in] ecode Code of the error to fill. If there is not enough memory for allocating error record then this code is
```

Comment:
This parameter is now [in,out]

Reply:
the function was changed according to @michalvasko comment